src/Microsoft.AspNet.Hosting/Program.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.AspNet.Hosting
 {
     public class Program
     {
-        private const string HostingIniFile = "Microsoft.AspNet.Hosting.ini";
+        private const string HostingJsonFile = "Microsoft.AspNet.Hosting.json";
```

Comment:
What is the point? The whole idea of switching to JSON is to make this consistent with where everything else is headed, so we might as well ditch the ini file now so that we can give people time to update their projects and then we can work on more pressing issues.